collector/src/main/java/com/bitmovin/analytics/adapters/ExoPlayerAdapter.java

```diff
@@ -272,19 +275,139 @@ private void decorateDataWithPlaybackInformation(EventData data) {
+    @Override
+    public void onPlayerError(EventTime eventTime, ExoPlaybackException error) {
+
+    }
+
+    @Override
+    public void onTracksChanged(EventTime eventTime, TrackGroupArray trackGroups, TrackSelectionArray trackSelections) {
+
+    }
+
+    @Override
+    public void onLoadStarted(EventTime eventTime, MediaSourceEventListener.LoadEventInfo loadEventInfo, MediaSourceEventListener.MediaLoadData mediaLoadData) {
+
+    }
+
+    @Override
+    public void onLoadCompleted(EventTime eventTime, MediaSourceEventListener.LoadEventInfo loadEventInfo, MediaSourceEventListener.MediaLoadData mediaLoadData) {
+
+    }
+
+    @Override
+    public void onLoadCanceled(EventTime eventTime, MediaSourceEventListener.LoadEventInfo loadEventInfo, MediaSourceEventListener.MediaLoadData mediaLoadData) {
+
+    }
+
+    @Override
+    public void onLoadError(EventTime eventTime, MediaSourceEventListener.LoadEventInfo loadEventInfo, MediaSourceEventListener.MediaLoadData mediaLoadData, IOException error, boolean wasCanceled) {
+
+    }
+
+    @Override
+    public void onDownstreamFormatChanged(EventTime eventTime, MediaSourceEventListener.MediaLoadData mediaLoadData) {
+    }
+
+    @Override
+    public void onUpstreamDiscarded(EventTime eventTime, MediaSourceEventListener.MediaLoadData mediaLoadData) {
+
+    }
+
+    @Override
+    public void onMediaPeriodCreated(EventTime eventTime) {
+
+    }
+
+    @Override
+    public void onMediaPeriodReleased(EventTime eventTime) {
+
+    }
+
+    @Override
+    public void onReadingStarted(EventTime eventTime) {
+
+    }
+
+    @Override
+    public void onBandwidthEstimate(EventTime eventTime, int totalLoadTimeMs, long totalBytesLoaded, long bitrateEstimate) {
+
+    }
+
+    @Override
+    public void onViewportSizeChange(EventTime eventTime, int width, int height) {
+
+    }
+
     @Override
-    public void onVideoEnabled(DecoderCounters counters) {
+    public void onNetworkTypeChanged(EventTime eventTime, @Nullable NetworkInfo networkInfo) {
 
     }
 
     @Override
-    public void onVideoDecoderInitialized(String decoderName, long initializedTimestampMs, long initializationDurationMs) {
+    public void onMetadata(EventTime eventTime, Metadata metadata) {
 
     }
 
     @Override
-    public void onVideoInputFormatChanged(Format format) {
-        Log.d(TAG, String.format("OnVideoInputFormatChanged: Bitrate: %d Resolution: %d x %d", format.bitrate, format.width, format.height));
+    public void onDecoderEnabled(EventTime eventTime, int trackType, DecoderCounters decoderCounters) {
+
+    }
+
+    @Override
+    public void onDecoderInitialized(EventTime eventTime, int trackType, String decoderName, long initializationDurationMs) {
+
+    }
+
+    @Override
+    public void onDecoderInputFormatChanged(EventTime eventTime, int trackType, Format format) {
+        Log.d(TAG, String.format("onDecoderInputFormatChanged: Bitrate: %d Resolution: %d x %d", format.bitrate, format.width, format.height));
```

Comment:
I guess this method is also called for audio tracks (see `trackType`). In the previous version you logged the `onAudioInputFormatChanged` event with following code line:
`Log.d(TAG, String.format("OnAudioInputFormatChnaged: Bitrate: %d MimeType: %s", format.sampleRate, format.sampleMimeType));`

However, this doesn't happen now anymore.

Reply:
yea, that method fires twice now, once for video and once for audio. @Tigraine would you prefer I exclude the audio quality change event?

```
05-18 13:48:07.086 22342-22342/com.bitmovin.exoplayeranalyticsexample D/ExoPlayerAdapter: onDecoderInputFormatChanged: Bitrate: 4800000 Resolution: 1920 x 1080
05-18 13:48:07.107 22342-22342/com.bitmovin.exoplayeranalyticsexample D/ExoPlayerAdapter: onDecoderInputFormatChanged: Bitrate: 128000 Resolution: -1 x -1
```

Reply:
No that's totally fine. We also send audio bitrate change events in analytics.

Reply:
@matamegger any other changes, or is this good to merge and release?

Reply:
Printing the Resolution for audio tracks seems odd for me. 
However, seems to be good for merge.